src/AirBench/Api/Controllers/ReviewController.cs

```diff
@@ -0,0 +1,192 @@
+        ///     "Id": `int`,
+        ///     "BenchId": `int`,
+        ///     "Description": `string`,
+        ///     "Rating": `int`
+        /// }
+        /// </summary>
+        /// <param name="id">The review ID.</param>
+        /// <returns>The response body.</returns>
+        [HttpGet]
+        public async Task<ReviewDetailsResponse> Details(int id)
+        {
+            var review = await _repo.GetAsync(id);
+            var response = new ReviewDetailsResponse();
+            if (review != null)
+            {
+                response.Success = true;
+                response.Id = review.Id;
+                response.BenchId = review.BenchId;
+                response.Description = review.Description;
+                response.Rating = review.Rating;
+            }
+            else
+            {
+                response.Success = false;
+            };
+            return response;
+        }
+
+        /// <summary>
+        /// Retrieves the list of all reviews.
+        /// 
+        /// ROUTE:
+        /// "review/list"
+        /// 
+        /// RESPONSE BODY:
+        /// {
+        ///     "Reviews": [
+        ///         {
+        ///             "BenchId": `int`,
+        ///             "Description": `string`,
+        ///             "Rating": `int`
+        ///         },
+        ///             .
+        ///             .
+        ///             .
+        ///     ]
+        /// }
+        /// </summary>
+        /// <param name="id">The review ID.</param>
+        /// <returns>The response body.</returns>
+        [HttpGet]
+        public async Task<ReviewListResponse> List()
+        {
+            var reviews = await _repo.ListAsync();
+            var response = new ReviewListResponse();
+            reviews.ForEach(r =>
+            {
+                var reviewInfo = new ReviewInfo()
+                {
+                    Id = r.Id,
+                    BenchId = r.BenchId,
+                    Description = r.Description,
+                    Rating = r.Rating
+                };
+                response.Reviews.Add(reviewInfo);
+            });
+            return response;
+        }
+
+        /// <summary>
+        /// Attempts to retrieve the list of all reviews for the bench with the given ID.
+        /// 
+        /// ROUTE:
+        /// "review/listfor/{id}"
+        /// 
+        /// RESPONSE BODY:
+        /// {
+        ///     "BenchId": {
+        ///     "Reviews": [
+        ///         {
+        ///             "BenchId": `int`,
+        ///             "Description": `string`,
+        ///             "Rating": `int`
+        ///         },
+        ///             .
+        ///             .
+        ///             .
+        ///     ]
+        /// }
+        /// </summary>
+        /// <param name="id">The bench ID.</param>
+        /// <returns>The response body.</returns>
+        [HttpGet]
+        public async Task<ReviewListForResponse> ListFor(int id)
```

Comment:
I am somewhat confused by creating a separate class just to encapsulate a List<Review> and BranchId, especially when Branch class itself already contains a List<Review> type object.